server.Tests/PathServices/SignalRRealtimeDataRepositoryTest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PathApi.Server.Tests.PathServices
+{
+    class SignalRRealtimeDataRepositoryTest
```

Comment:
Can you add a TODO() for this test coverage to be implemented?

We can do this in a follow-up.